cfg/VideoStream.cfg

```diff
@@ -23,6 +23,7 @@ gen.add("flip_horizontal", bool_t, LEVEL.NORMAL, "Flip image horizontally", Fals
 gen.add("flip_vertical", bool_t, LEVEL.NORMAL, "Flip image vertically", False)
 gen.add("width", int_t, LEVEL.RUNNING, "Target width", 0, 0, 10000)
 gen.add("height", int_t, LEVEL.RUNNING, "Target height", 0, 0, 10000)
+gen.add("set_camera_properties", bool_t, LEVEL.RUNNING, "Set the camera properties (brightness, contrast, hue, saturation)", False)
 gen.add("brightness", double_t, LEVEL.RUNNING, "Target brightness", 0.5019607843137255, 0.0, 1.0)
```

Comment:
This could be better if it is "Target brightness (Available only when `set_camera_properties` is true)".
Same on the "contrast", "hue" and "saturation".